app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers/topics.controller");
+const {handlePsqlErrors ,handleCustomErrors}= require("./errors");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
At this point express.json isn't required. Please remove this until you get to the point you need it 🙂

Reply:
ok I will remove it in the next pr